easybuild/easyconfigs/s/SpFFT/SpFFT-0.9.8-CrayGNU-19.10-cuda-10.1.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'CMakeMake'
+
+name = 'SpFFT'
+version = '0.9.8'
+cudaversion = 10.1
+versionsuffix = '-cuda-%s' % cudaversion
+
+homepage = 'https://github.com/eth-cscs/SpFFT'
+description = """Sparse 3D FFT library with MPI, OpenMP, CUDA and ROCm support"""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.10'}
+toolchainopts = {'usempi': True, 'openmp': True, 'pic': True, 'verbose': False, 'opt': True}
+
+source_urls = ['https://github.com/eth-cscs/SpFFT/archive']
+sources = ['v%(version)s.tar.gz']
+
+builddependencies = [('CMake', '3.14.5', '', True)]
+
+dependencies = [
+  ('cudatoolkit' , EXTERNAL_MODULE),
+  ('gcc', EXTERNAL_MODULE),
```

Comment:
`gcc` is already loaded by the CrayGNU toolchain, is this really needed?

Reply:
I've removed it.